topics/introduction/slides/index.html

```diff
@@ -184,6 +186,14 @@
 
 ![Workflow interface](../images/RNA-Seq_workflow_editing.png)
 
+???
```

Comment:
Is that intentional?

Reply:
Yes, it's the syntax to start the speaker notes.

Reply:
oh ok, did not know that.